static/src/stylesheets/inline/article-labour-liverpool.scss

```diff
@@ -259,3 +259,70 @@ $labour-liverpool-header-element-opacity: .85;
         height: 88%;
     }
 }
+.content__article-body {
+    figure {
+        &.element-video--heroic {
```

Comment:
@SiAdcock if I un nest it does seem to go a bit mental. As there are classes elsewhere which are overriding. As they are more heavily nested.